src/components/Button/Button.tsx

```diff
@@ -2,10 +2,14 @@ import { Icon } from "@/components";
 import { IconName } from "@/components/Icon/types";
 import styled from "styled-components";
 
-type ButtonType = "primary" | "secondary";
-type Alignment = "left";
+type ButtonType = "primary" | "secondary" | "danger";
+type ButtonState = "default" | "hover" | "active";
+type Alignment = "default" | "left";
+
 export interface ButtonProps extends React.HTMLAttributes<HTMLButtonElement> {
   type?: ButtonType;
+  state?: ButtonState;
+  isDisabled?: boolean;
```

Comment:
i think everywhere else we went for the original `disabled` property, maybe we can keep it here too to make it more consistent

Reply:
Fixed